articles/getting-started-using-admob-with-react-native-expo/index.md

```diff
@@ -0,0 +1,491 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/getting-started-using-admob-with-react-native-expo/
+title: Getting started using Admob to React Native Expo 
+description: In this tutorial we are going to go over how to set up and configure Admob into a React Native app based on the Expo ecosystem.
+author: worawat-kaewsanmaung
+date: 2020-12-08T00:00:00-11:00
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-using-admob-with-react-native-expo/hero.jpg
+    alt: React Admob with Expo
+---
+Many modern-day apps contain ads in them. The ads pop out if you want to use certain features in the app. Mainly in game apps, you need to watch an ad before reaching another level. Ads have been a popular way to monetize the app and make some profit out of it. Some sophisticated apps also provide a payment option to remove the ads from the interface. Here, we are going to incorporate the same logic but in a slightly different way.
+<!--more-->
+In this tutorial, we are going to implement Admob in the React Native application. Here, we are going to implement an interface to show the ads as well as an option to remove ads as well. In most cases, the users will have to pay to remove the ads. But here, we are going to make users watch video ads to hide the ads from the interface for a certain amount of time. For this, we are going to make use of the expo Admob package that provides the ad reward feature.
+
+### Requirement
+- A fresh React Native project or download the [starter project](https://snack.expo.io/2y6z09xog) from the [previous tutorial](/engineering-education/how-to-control-dark-mode-in-react-native-using-redux/).
+- It is necessary to complete the  before starting this one. The previous tutorial will help you understand the use of React Navigation. Also, the use of React Native Paper components, and the Redux mechanism better. You can use the code from the previous tutorial as a for this tutorial.
```

Comment:
Since you have provided the starter code, it isn't actually necessary to go through the previous tutorial. 
Phrase it as the previous tutorial is optional if they want to learn how you built the starter code.

Also, just say download the starter code. No need for A fresh React Native project, since you will be using the starter code primarily in the tutorial.